cfme/containers/image.py

```diff
@@ -67,8 +57,8 @@ def navigate(self, detail=True):
             if not self._on_detail_page():
                 sel.force_navigate(
                     'containers_image_detail', context={
-                        'image': self, 'provider': self.provider})
```

Comment:
No. The thing is that the nav steps were badly designed. They asked for 2 things despite one thing providing the other :) So I removed that and made the nav steps use the dependency.